Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -254,11 +249,9 @@ public async Task InstallPackage(PackageVersionInfo package, CancellationToken c
 
             var installationInfo = new InstallationInfo
             {
-                Id = Guid.NewGuid(),
+                Guid = package.guid,
```

Comment:
if you're calling a field `Guid` you should probably store a real Guid object there, not its string representation

Reply:
It works the same way in the other package objects and I didn't want to update every reference to use a read GUID throughout the project right now. This won't affect the manifest though so we can always change it later.